atc/exec/get_step.go

```diff
@@ -163,6 +139,11 @@ func (step *GetStep) Run(ctx context.Context, state RunState) error {
 		ResourceTypes: resourceTypes,
 	}
 
+	imageSpec := worker.ImageFetcherSpec{
+		ResourceTypes: resourceTypes,
+		Delegate:      step.delegate,
+	}
+
 	resourceCache, err := step.resourceCacheFactory.FindOrCreateResourceCache(
```

Comment:
at this point what's stopping us from moving the `resource_cache` over to the runtime side?